iwyu_path_util.cc

```diff
@@ -97,20 +97,28 @@ string CanonicalizeFilePath(const string& path) {
 }
 
 string CanonicalizeHeaderSearchPath(const string& path) {
-  string result = CanonicalizeFilePath(path);
+  string result = path;
+  // Header search paths must be absolute
+  if (!IsAbsolutePath(result))
```

Comment:
Please collapse this to:

    result = MakeAbsolutePath(path);

I think it should amount to the same thing.